site/content/Getting Started/_index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Getting Started"
+linkTitle: "Getting Started"
+weight: 2
+description: >
+    Taking VHS for a spin.
+---
+
+The quickest way to see `vhs` in action is to use the development Docker image in this repo to run a demo.
```

Comment:
Since this is largely a copy of what's in the readme, I wonder if we should just have the readme link to this page instead

Reply:
I think there might be some value in having a super simple demo for the README, but for now I've changed this to just link to the getting started page on the docs site.